TouchEffectSample.iOS/TouchEffectSample.iOS.csproj

```diff
@@ -148,7 +148,7 @@
   </ItemGroup>
```

Comment:
the file should be discarded

Reply:
I think I'm missing something, but why? Wouldn't this break the iOS Sample project?